label_studio/projects/models.py

```diff
@@ -890,9 +890,7 @@ def max_tasks_file_size():
     def get_parsed_config(self, autosave_cache=True):
```

Comment:
Since this appears to be having no effect right now with `autosave_cache` do we want to remove this parameter and remove the only False case used:

label_studio/projects/models.py|699 col 24| outputs = self.get_parsed_config(autosave_cache=False)

Reply:
done